### DISCUSSION TITLE
Can Organicmaps record tracks ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [organicmaps/organicmaps#8678] [Q&A]
Hi everyone,

in `Bookmarks and Tracks` we can _EXPORT ALL BOOKMARKS AND TRACKS_

Tracks related to what ? Can we record track with Organicmaps ? 

Thanks.

**Comment by biodranik** [comment]
Now you can import KML, KMZ, GPX tracks into Organic Maps. We're working on track recording now.

**Reply by BobSquarePants** [reply]
Thanks @biodranik ,
I don't know if you should develop a build-in track recorder.. ?
I'm using [OpenTracks](https://github.com/OpenTracksApp/OpenTracks) and it works well with Organicmaps. !

Cheers
